packages/contracts-bedrock/test/L1/BatchInbox.t.sol

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+// Testing
+import { Test } from "forge-std/Test.sol";
+
+// Contracts
+import { BatchInbox } from "src/L1/BatchInbox.sol";
+import { IBatchAuthenticator } from "interfaces/L1/IBatchAuthenticator.sol";
+
+/// @title MockBatchAuthenticator
+/// @notice Mock implementation for testing - only implements validBatchInfo
+contract MockBatchAuthenticator {
+    mapping(bytes32 => bool) private validHashes;
```

Comment:
Nit: `hashValidities` might be a more accurate name since the mapped `bool` might be `false`.